roles/build-release/README.md

```diff
@@ -4,12 +4,8 @@ An Ansible role that builds an Ansible release package with Antsibull and tests
 
 ### Requirements
 
-This role is expected to run directly from an antsibull git repository checkout.
```

Comment:
Where would the the role be if not in the git repo ? Should we consider moving it to it's own repo/collection ?